src/streaming/utils/TTMLParser.js

```diff
@@ -71,6 +73,133 @@ function TTMLParser() {
     }
 
     /**
+     * Get the begin-end interval if present, or null otherwise.
+     *
+     * @param {Object} element - TTML element which may have begin and end attributes
+     */
+    function getInterval(element) {
+        if (element.hasOwnProperty('begin') && element.hasOwnProperty('end')) {
```

Comment:
I mean the ISOBMFF sample in which this document was delivered, which has its own composition time and duration. Thinking about my comments further, I have probably over-simplified due to lack of understanding of the code - apologies. The composition time begin and end values would apply if, working up from a leaf element, no element in that part of the document hierarchy has begin or end values. However I think that case is already dealt with by the document activation rules (i.e. zero or one document active at any given time, as discussed some time ago).

Sorry for the noise!